CONTRIBUTING.md

```diff
@@ -65,8 +189,49 @@ please ensure that:
 4. Ensure that if your change has an end user facing impact (new feature,
```

Comment:
In step 2, just above in the folded bits of the diff, there's a link to `qiskit.org/docuemntation/contributing_to_qiskit.html#documentation-structure`, which I think might need changing?

Reply:
looks like that link doesn't even exist in our current qiskit.org page, i'm guessing it was from an old old version of the contributing page. I think safest to just remove